user_docs/en/userGuide.t2t

```diff
@@ -3659,11 +3664,11 @@ Due to this, and to maintain compatibility with other screen readers in Taiwan,
 
 ++ Eurobraille displays ++[Eurobraille]
 The b.book, b.note, Esys, Esytime and Iris displays from Eurobraille are supported by NVDA.  
-These devices have a braille keyboard with 10 keys. 
+These devices have a Braille keyboard with 10 keys. 
 Of the two keys placed like a space bar, the left key is corresponding to the backspace key and the right key to the space key.
-Connected via USB, these devices have one stand-alone usb keyboard. 
-It is possible to enable/disable this keyboard with the checkbox ‘HID Keyboard simulation’ in braille setting panel. 
-The braille keyboard describes below is the braille keyboard when this checkbox is not checked.

```

Comment:
This sentence also needs clarification as per [this thread](https://groups.io/g/nvda-translations/topic/line_3617_and_3619/100311411?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,100311411,previd%3D1690294920572050656,nextid%3D1654591085060908112&previd=1690294920572050656&nextid=1654591085060908112).

Probably at least changing "describes" to "described"?

Reply:
I've opened #15194 to address this